src/refactor_nrepl/core.clj

```diff
@@ -65,18 +65,28 @@
   Should generally always be true, except in a few refactor-nrepl's unit tests."
   true)
 
+(def ^:private resources-dir-pattern
+  (re-pattern (str "resources" File/separator "?$")))
+
+(def ^:private target-dir-pattern
+  (re-pattern (str "target" File/separator "?$")))
+
+(defn irrelevant-dir?
+  "Does `f` look like a directory without files that should be analyzed/refactored?"
+  [^File f]
+  (let [s (-> f .toString)]
+    (boolean (or (if *skip-resources?*
```

Comment:
I'd probably skip this conversion to boolean, but I'm fine either way. I like boolean predicates, but I know most of the community doesn't care what exactly gets returned.

Reply:
Yeah grey zone, for me the decider was leaving an expressive unit test